integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -282,7 +287,19 @@ def move(self, target, pos=None):
         # Moving a node can modify all other nodes via raw sql queries (which are not recognized by cachalot),
         # so we have to invalidate the whole model manually.
         invalidate_model(self.__class__)
-        super().move(target=target, pos=pos)
+        try:
+            with db_mutex("page-tree"):
+                super().move(target, pos)
+        except DBMutexError as e:
+            logger.error("Could not retrieve database mutex to save %s", self)
+            raise DBMutexError(
+                _('Could not change position in tree of "{}".').format(self)
```

Comment:
```suggestion
                _('Could not change position in tree of "{}".').format(self.best_translation.title)
```
For sub-pages, this results in:

> Could not change position in tree of "Willkommen \&rarr; Stadtplan".

because since #2016 the string-repreesentation was changed into the path instead the title (which I would revert at some point, but right now I'd also be more explicit how the page is represented, especially since `&rarr;` is printed literally and not rendered to an arrow).

Reply:
Ah sorry, in this case I missed that the language tree nodes do not have a translation... So maybe we have to settle for the default string representation anyways...

Reply:
I like the idea of having readable titles. But at the same time I think it is highly unlikely that any user will ever see this message :D So yeah, lets just use %r.